python/ray/air/config.py

```diff
@@ -171,12 +172,16 @@ def from_placement_group_factory(
             num_workers = len(worker_bundles)
             resources_per_worker = first_bundle
 
+        if "_max_cpu_fraction_per_node" in pgf._kwargs:
+            max_cpu_fraction_per_node = pgf._kwargs["_max_cpu_fraction_per_node"]
+
         return ScalingConfigDataClass(
             trainer_resources=trainer_resources,
             num_workers=num_workers,
             use_gpu=use_gpu,
             resources_per_worker=resources_per_worker,
             placement_strategy=placement_strategy,
+            _max_cpu_fraction_per_node=max_cpu_fraction_per_node,
```

Comment:
Bug where this wasn't propagated to copies of the Trainer. Didn't affect correctness, but broke the warning.